src/c++/perf_analyzer/infer_data_manager_base.cc

```diff
@@ -30,6 +30,89 @@
 
 namespace triton { namespace perfanalyzer {
 
+cb::Error
+InferDataManagerBase::GetInputData(
```

Comment:
Is this just a copy and paste? If so, where from? If not, was it refactored to keep the same external functionality? Or is this a new design?

Reply:
Copy and paste refactor. It wasn't strictly copy and paste -- the shm and non-shm paths had slightly different code organization that did the same thing.